src/microbe_stage/Microbe.Interior.cs

```diff
@@ -1416,7 +1416,7 @@ private void HandleDigestion(float delta)
                 var amount = MicrobeInternalCalculations.CalculateDigestionSpeed(Enzymes[usedEnzyme]);
                 amount *= delta;
 
-                // Efficiency starts from 40% up to 60%. This means at least 7 lysosomes are needed to achieve
+                // Efficiency starts from 30% up to 60%. This means at least 7 lysosomes are needed to achieve
```

Comment:
I think for future proofing this comment it might be good to say that check the actual constants / code for the real values. Or optionally all numbers could be removed from this comment.